### ISSUE TITLE
Compass Display is Incorrect by 180 deg

### ISSUE THREAD

**Issue by jesawyers** [nitroman123/Ship-Simulator-Realistic#19]
Exiting Syros with mini map **in north up mode.**

Based on the mini map my heading should be around 115 deg.

The compass is showing an error of 180 degrees.  If you take the displayed compass heading of 295 and subtract 180 you get the correct  heading of 115 degrees.

![image](https://user-images.githubusercontent.com/13039522/147789402-3e56666f-f958-412b-a6a8-2c32296d43ac.png)

**Comment by nitroman123** [comment]
when the north up is selected the compass doesnt change i think it shows correct but not sure.

**Comment by jesawyers** [comment]
Compass is incorrect.  If the mini map is north up, then the mini map is showing the correct direction the ship is pointed.  This means 090 is to the right on the mini map, 180 is down,  270 is to the left.  Based on this and how the ship is oriented on the mini map, the ships heading is around 115 deg.

**Comment by jesawyers** [comment]
Yes I agree that when in north up mode that the compass does not change.

![image](https://user-images.githubusercontent.com/13039522/147881037-c2782fd5-b83b-46a3-85c7-1fb51e7e1e3a.png)

**Comment by jesawyers** [comment]
This was done on the Full screen map which is always in north up mode.

**Comment by nitroman123** [comment]
> This was done on the Full screen map which is always in north up mode.

thats correct so i need to divide by 180 ?

**Comment by jesawyers** [comment]
Subtracting 180 **in this situation** will give the correct heading.  

If you simply subtract 180 from your current calculation, you will need to check to see if the new heading is less that 000 or >= 359.99999 (360).  

Simply subtracting 180 from your current calculation could give you values not between 000 or 359.9999 (360).
